Resources/Prototypes/Entities/Clothing/Neck/base_clothingneck.yml

```diff
@@ -16,6 +16,9 @@
     spawned:
     - id: MaterialCloth1
       amount: 2
+  - type: PhysicalComposition # 50% of the amount of cloth you'd obtain upon butchering
+    materialComposition:
+      Cloth: 100
```

Comment:
Ok, for some reason the crew medals, pins and headphones inherit this and are butcherable for that reason. There should probably be a ClothingNeckBaseButcherable prototype to fix that, but that is not your fault can be done in another PR